ltr/concourse/deploy.py

```diff
@@ -43,6 +45,19 @@
     update_endpoint=True,
 )
 
+# wait for the deployment to complete
+endpoint_status = "Updating"
+while endpoint_status == "Updating":
```

Comment:
Do you think checking if `endpoint_status != OK` (replacing OK with whatever the OK status is) would help? I'm wondering if there are other statuses that aren't OK during which we'd also want to wait.

Reply:
I think we want to bail out when it's not `Updating`.  There's a list of them here: https://docs.aws.amazon.com/cli/latest/reference/sagemaker/describe-endpoint.html

`Creating` won't happen (outside the bootstrap job), `SystemUpdating` means AWS maintenance is happening and we don't know how long it'll take (and if we're checking for that we should do it before even trying to deploy the model), and `RollingBack` will get back to the old working configuration successfully if we don't delete it.

Reply:
OK, thanks for that detail!